core/src/main/java/com/redhat/lightblue/client/response/LightblueResponse.java

```diff
@@ -40,13 +42,56 @@ public void setJson(JsonNode json) {
         this.json = json;
     }
 
-    public boolean hasError(){
+    public boolean hasError() {
         JsonNode objectTypeNode = json.get("status");
-        if(objectTypeNode == null){
+        if (objectTypeNode == null) {
             return false;
         }
 
         return objectTypeNode.textValue().equalsIgnoreCase("error");
     }
 
+    public int parseModifiedCount() {
+        return parseInt("modifiedCount");
+    }
+
+    public int parseMatchCount() {
+        return parseInt("matchCount");
+    }
+
+    private int parseInt(String fieldName) {
+        JsonNode field = json.findValue(fieldName);
+        if (field == null || field.isNull()) {
+            return 0;
+        }
+        return field.asInt();
+    }
+
+    public <T> T parseProcessed(final Class<T> type) throws LightblueResponseParseException {
+        return parseProcessed(new ObjectMapper(), type);
```

Comment:
Shouldn't a better configured ObjectMapper be created for parsing JSON?  For example, see https://github.com/lightblue-platform/lightblue-client/blob/85c392811d6b73b250c99438771c99ea97dea5a6/http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java#L44-L46